.github/workflows/release.yml

```diff
@@ -93,9 +93,10 @@ jobs:
           architecture: ${{ matrix.arch }}
       - name: Setup Ant
         uses: cedx/setup-ant@v3
-      - name: Write to todo.txt
+      - name: Write to `todo.txt` and update `Base.java`
```

Comment:
I'd suggest a clearer name like "Write revision number to `todo.txt` and `Base.java`"